pkg/tfbridge/config_encoding.go

```diff
@@ -204,9 +181,15 @@ func (enc *configEncoding) UnmarshalProperties(props *structpb.Struct,
 }
 
 // Inverse of UnmarshalProperties, with additional support for top-level (but not nested) secrets.
```

Comment:
I thought we now support nested secrets.

Reply:
Updated comment, obsolete comment.